crates/nu-command/tests/commands/into_int.rs

```diff
@@ -31,6 +31,13 @@ fn into_int_binary() {
     assert!(actual.out.contains("16843009"));
 }
 
+#[test]
+fn into_int_binary_signed() {
+    let actual = nu!("echo 0x[f0] | into int --signed");
+
+    assert!(actual.out.contains("-16"));
+}
+
```

Comment:
Maybe worth adding a round-trip test both for `--signed` the default `into int` going back to `into binary`